_partials/_cloud-intro.md

```diff
@@ -5,13 +5,11 @@ scale further, and stay under budget. Timescale Cloud offers the following Postg
   [time-series and analytics][what-is-time-series] workloads. Get automated dynamic data partitioning, hybrid 
   row-columnar storage, advanced compression techniques, incremental up-to-date materializations, and specialized 
   analysis functions as well as cloud-only features like transparent tiering and low-cost object storage.
-- **[Dynamic PostgreSQL][create-service]**: for all other workloads. A PostgreSQL instance with a
-  [dynamic compute range][what-is-dynamic-postgres] aligned to your business needs. You select a compute range, only 
+- **[Dynamic PostgreSQL][create-service]**: for all other workloads. A PostgreSQL instance with a dynamic compute range aligned to your business needs. You select a compute range, only 
```

Comment:
I thought the point was to remove mentions of Dynamic Postgres. Can you have a check through and update to do:
* Time series
* Vector and AI
* Postgres
And only leave a mention of dynamic for the get started where that is still in the UI.